Examples/VaporExample/README.md

```diff
@@ -43,19 +43,14 @@ If you'd like to point the application to a MongoDB server elsewhere (e.g. on [M
 
 The call to `configure()` initializes a global `MongoClient` to back your application. `MongoClient` is implemented with that approach in mind: it is safe to use across threads, and is backed by a [connection pool](https://en.wikipedia.org/wiki/Connection_pool) which enables sharing resources throughout the application.
 
-Throughout your application, you can access the global client via `app.mongoDB.client`. Note that the global client may return `EventLoopFuture`s on *any* `EventLoop` in the application's `EventLoopGroup`, so if you use this client you will need to ensure you "hop" the futures back to the event loop you are currently on. See the `EventLoopFuture` [documentation](https://apple.github.io/swift-nio/docs/current/NIO/Classes/EventLoopFuture.html) for more details.
```

Comment:
I suspected, and confirmed with folks on the Vapor discord server, that the whole "stay on the same event loop that the request is executing on" requirement from the future based API is gone when using async/await, since the request handler closures are not running on event loops anymore but instead on `Task`s.

given that, I don't think that there is a reason for async/await users to use the "event loop bound" clients/DBs/collections we added support for last year. I don't think it is particularly harmful for users to do this -- it just means that anytime we cross into EventLoop land from the Request's `Task` we would be on the same `EventLoop` -- but there's also no real benefit, and there might even be some performance impacts by constraining things to particular event loops. So I've updated the example here to just direct users to the global client instead, and for 2.0 of MongoDBVapor we'd maybe either get rid of the extension on `Request` or update it so that it just points to the global client.

I filed SWIFT-1449 about these changes